### PR TITLE
Add the current .scalafmt.conf

### PR DESCRIPTION
**Pull request by chick**
I plan to start using this on new code and eventually on the complete repo
Should help uncover problems we may encounter moving to this.

### CODE REVIEW

**Comment by chick** [comment]
Everything in source has been reformatted.
@seldridge, @azidar, @ucbjrl,, @jackkoenig, @ducky64 
You might want to take a look at this. Because treadle has no other PRs at the moment (other than this one) it is an easier repo to use.

**Comment by chick** [comment]
@ucbjrl BTW, I added a short section on formatting to readme

File: src/main/scala/treadle/executable/ClockStepper.scala [hunk 1]
```diff
@@ -30,20 +30,20 @@ class NoClockStepper extends ClockStepper {
 case class ClockAssigners(upAssigner: Assigner, downAssigner: Assigner)
 
 case class SimpleSingleClockStepper(
-  engine: ExecutionEngine,
-  dataStore: DataStore,
-  clockSymbol: Symbol,
-  resetSymbolOpt: Option[Symbol],
-  clockPeriod: Long,
+  engine:             ExecutionEngine,
```
Comment (ducky64) [inline src/main/scala/treadle/executable/ClockStepper.scala]: Not a huge fan of this, but not blocking either

**Comment by chick** [comment]
@ucbjrl Can you take a look at the README and let me know if it needs more than this

**Pull request merged**